src/main/java/com/google/firebase/auth/FirebaseUserManager.java

```diff
@@ -204,36 +184,61 @@ DownloadAccountResponse listUsers(int maxResults, String pageToken) throws Fireb
       builder.put("nextPageToken", pageToken);
     }
 
-    DownloadAccountResponse response;
-    try {
-      response = post("downloadAccount", builder.build(), DownloadAccountResponse.class);
-      if (response == null) {
-        throw new FirebaseAuthException(LIST_USERS_ERROR,
-            "Unexpected response from download user account API.");
-      }
-      return response;
-    } catch (IOException e) {
-      throw new FirebaseAuthException(LIST_USERS_ERROR,
-          "IO error while downloading user accounts.", e);
+    DownloadAccountResponse response = post(
+        "downloadAccount", builder.build(), DownloadAccountResponse.class);
+    if (response == null) {
+      throw new FirebaseAuthException(INTERNAL_ERROR,
+          "Unexpected response from download user account API.");
```

Comment:
Do you want to match you other error messages and say "Failed to retrieve..."?

Reply:
Done